### PR TITLE
matching test-harness when casc is added

### PR DESCRIPTION
**Pull request by joseblas**

### CODE REVIEW

**Comment by jglick** [comment]
Is this superseded by https://github.com/jenkinsci/bom/pull/315?

**Comment by timja** [comment]
> Is this superseded by [jenkinsci/bom#315](https://github.com/jenkinsci/bom/pull/315)?

looks like not:
https://github.com/jenkinsci/bom/pull/294#issuecomment-710554319

**Comment by timja** [comment]
Tested this in https://github.com/jenkinsci/bom/pull/294 and it works...

**Comment by jglick** [comment]
Please merge with master @joseblas.

File: plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java [hunk 1]
```diff
@@ -379,11 +381,11 @@ public PluginCompatReport testPlugins()
         }
 
         if (failed && config.isFailOnError()) {
-            throw new AbortException("Execution was aborted due to the failure in a plugin test (-failOnError is set)");
+		    throw new AbortException("Execution was aborted due to the failure in a plugin test (-failOnError is set)");
```
Comment (timja) [inline plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java:384]: unnecessary whitespace change, and adds tabs to a file that uses spaces

File: plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java [hunk 2]
```diff
@@ -379,11 +381,11 @@ public PluginCompatReport testPlugins()
...
         }
 
         return report;
-    }
+	}
```
Comment (timja) [inline plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java:388]: unnecessary whitespace change, and adds tabs to a file that uses spaces

File: plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java [hunk 3]
```diff
@@ -432,7 +434,7 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
 
         File pluginCheckoutDir = new File(config.workDirectory.getAbsolutePath() + File.separator + plugin.name + File.separator);
 
-        try {
+		try {
```
Comment (timja) [inline plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java:437]: unnecessary whitespace change, and adds tabs to a file that uses spaces

**Comment by timja** [comment]
@joseblas why was this closed? we're relying on it in bom

**Comment by jglick** [comment]
If we are consuming it, it needs to remain open.

**Comment by jglick** [comment]
Replaced by #276.
